File: code/connect.php
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>地區選擇</title>
</head>
<body>

<?php
$host = 'localhost';
//改成你登入phpmyadmin帳號
$user = 'root';
//改成你登入phpmyadmin密碼
$passwd = '123456';
//資料庫名稱
$database = 'test';
//實例化mysqli(資料庫路徑, 登入帳號, 登入密碼, 資料庫)
$connect = new mysqli($host, $user, $passwd, $database);
 
if ($connect->connect_error) {
    die("連線失敗: " . $connect->connect_error);
}
echo "連線成功";

$connect->query("SET NAMES 'utf8'");
 
//選擇資料表user
$selectSql = "SELECT * FROM names";
//呼叫query方法(SQL語法)
$memberData = $connect->query($selectSql);
//有資料筆數大於0時才執行
if ($memberData->num_rows > 0) {
//讀取剛才取回的資料
    while ($row = $memberData->fetch_assoc()) {
        print_r($row);
    }
} else {
    echo '0筆資料';
}

?>


</body>
</html>

File: code/connect.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import MySQLdb

db = MySQLdb.connect( 
	host="localhost",     #主機名
	user="root",          #用戶名
	passwd="sam8617527",   #密碼
	db="test")         #數據庫名稱

#查詢前，必須先獲取游標
cur = db.cursor()

#執行的都是原生SQL語句
cur.execute("SELECT * FROM names")

for row in cur.fetchall(): 
    print (row[0])

db.close()
